components/atom/button/src/index.scss

```diff
@@ -175,6 +175,7 @@ $p-atom-button-large: $p-l !default;
 
   &--fullWidth {
     justify-content: center;
```

Comment:
I think that this property is being ignored because the element is `display: inline-block` but I rather do not remove this in order to avoid possible problems. :) We might need to investigate further before removing it.